### DISCUSSION TITLE
Image colors do not match Windows Viewer

### DISCUSSION THREAD

**Discussion by Mac3DX** [QL-Win/QuickLook#1255] [General]
It seems not to be accurate with colors visualization also. In Windows viewer the colors look perfect. It seems not to be loading some color profile I guess, I'm not sure but the colors are off.

**Comment by rabelux** [comment]
QuickLook doesn't respect your monitor profile by default. To enable it have a look here: https://github.com/QL-Win/QuickLook/wiki/Advanced-configurations#enabledisable-monitor-color-profile-conversion-since-v317-6

**Reply by Mac3DX** [reply]
Hey! Awesome, I didn't notice any drop over performance. Thanks!

**Reply by Mac3DX** [reply]
Has this feature stopped working in Windows 23H2? I’m using a color-profiled P3 monitor, and it can’t properly display sRGB images -they look oversaturated.
